client/client-multi/client-cpp/Modules/FindBotan.cmake

```diff
@@ -20,115 +20,108 @@
 # This file defines:
 # * BOTAN_FOUND if protoc was found
 # * BOTAN_LIBRARY The lib to link to (currently only a static unix lib, not
-# portable) 
+# portable)
 # * BOTAN_INCLUDE The include directories for libbotan.
 
-message(STATUS "FindBotan check")
-IF (NOT WIN32)
-  include(FindPkgConfig)
-  if ( PKG_CONFIG_FOUND )
+message(status "FindBotan check")
 
-     pkg_check_modules (PC_BOTAN botan-1.11)
+if(NOT WIN32)
+    include(FindPkgConfig)
+    if(PKG_CONFIG_FOUND)
+        pkg_check_modules(PC_BOTAN botan-1.11)
+        set(BOTAN_DEFINITIONS ${PC_BOTAN_CFLAGS_OTHER})
+    endif(PKG_CONFIG_FOUND)
+endif(NOT WIN32)
 
-     set(BOTAN_DEFINITIONS ${PC_BOTAN_CFLAGS_OTHER})
-  endif(PKG_CONFIG_FOUND)
-endif (NOT WIN32)
-
-#
 # set defaults
-SET(_botan_HOME "/usr/local")
-SET(_botan_INCLUDE_SEARCH_DIRS
-  ${CMAKE_INCLUDE_PATH}
-  /usr/local/include
-  /usr/include
-  )
-
-SET(_botan_LIBRARIES_SEARCH_DIRS
-  ${CMAKE_LIBRARY_PATH}
-  /usr/local/lib
-  /usr/lib
-  )
-
-##
-if( "${BOTAN_ROOT}" STREQUAL "")
-  if("" MATCHES "$ENV{BOTAN_ROOT}")
-    message(STATUS "BOTAN_ROOT env is not set, setting it to /usr/local")
-    set (BOTAN_ROOT ${_botan_HOME})
-  else("" MATCHES "$ENV{BOTAN_ROOT}")
-    set (BOTAN_ROOT "$ENV{BOTAN_ROOT}")
-  endif("" MATCHES "$ENV{BOTAN_ROOT}")
-else( "${BOTAN_ROOT}" STREQUAL "")
-  message(STATUS "BOTAN_ROOT is not empty: \"${BOTAN_ROOT}\"")
-endif( "${BOTAN_ROOT}" STREQUAL "")
-##
+set(_botan_HOME "/usr/local")
+set(_botan_INCLUDE_SEARCH_DIRS
+    ${CMAKE_INCLUDE_PATH}
+    /usr/local/include
+    /usr/include)
+
+set(_botan_LIBRARIES_SEARCH_DIRS
+    ${CMAKE_LIBRARY_PATH}
+    /usr/local/lib
+    /usr/lib)
```

Comment:
Please, remove `/usr/local/lib` and `/usr/lib`. We should use `${CMAKE_SYSTEM_LIBRARY_PATH}` and `${CMAKE_SYSTEM_INCLUDE_PATH}` for include directories. (We don't have to do anything, as that's default for `find_library`, `find_file` and `find_path`.) Not respecting these variables will lead to horrible-horrible debugging during cross-compilation.